oauth2-ballerina/build.gradle

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+description = 'Ballerina - OAuth2 Ballerina'
+
+configurations {
+    jbalTools
+    externalJars
```

Comment:
```suggestion
```

Do we need this?

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-oauth2/pull/1/commits/1bd7b2fb1dbc4d3fe2630d560777e28865c03bf4